docs/Installation.md

```diff
@@ -202,191 +202,130 @@ The command is expected to give an immediate JSON response similiar to :
 
 ## 2. Installation on Debian
 
-### 2.1 Installing dependencies
+### 2.1 Install Zonemaster::Backend and related dependencies (Debian)
 
-```sh
-sudo apt-get update
-sudo apt-get install git libmodule-install-perl libconfig-inifiles-perl libdbd-sqlite3-perl starman libio-captureoutput-perl libproc-processtable-perl libstring-shellquote-perl librouter-simple-perl libclass-method-modifiers-perl libtext-microtemplate-perl libdaemon-control-perl
-sudo cpan -i  Test::Requires Plack::Middleware::Debug Parallel::ForkManager JSON::RPC 
-```
->
-> Note: The Perl modules `Parallel::ForkManager` and `JSON::RPC` exist as Debian
-> packages, but with versions too old to be useful for us.
->
-
-### 2.2 Install the chosen database engine and related dependencies
-
-#### 2.2.1 MySQL
+Install dependencies available from binary packages:
 
 ```sh
-sudo apt-get install mysql-server libdbd-mysql-perl
+sudo apt-get install libclass-method-modifiers-perl libconfig-inifiles-perl libdaemon-control-perl libdata-dump-perl libdbd-sqlite3-perl libdbi-perl libfile-sharedir-perl libfile-slurp-perl libhtml-parser-perl libintl-perl libio-captureoutput-perl libjson-pp-perl libjson-rpc-perl libmodule-build-tiny-perl libmoose-perl libplack-perl librouter-simple-perl libstring-shellquote-perl libtest-requires-perl libtest-warn-perl libtext-microtemplate-perl libtie-simple-perl starman
 ```
 
-#### 2.2.2 PostgreSQL
+Install dependencies not available from binary packages:
 
 ```sh
-sudo apt-get install libdbd-pg-perl postgresql
+sudo cpan -i Plack::Middleware::Debug Parallel::ForkManager Net::IP::XS
 ```
 
-#### 2.2.3 SQLite
+> **Note:** Zonemaster::LDNS and Zonemaster::Engine are not listed here as they
+> are dealt with in the [prerequisites](#prerequisites) section.
 
->
-> At this time there is no instruction for using SQLite on Debian and Ubuntu.
->
+> **Note:** The Perl modules `Parallel::ForkManager` and `JSON::RPC` exist as

```

Comment:
JSON::RPC should be listed in Makefile.PL with required version.

Is the comment really needed here? If Makefile.PL lists required version then that is handled when installation of Backend from CPAN is done.

Reply:
Done.